src/core_modules/capture-core/components/WidgetEventComment/WidgetEventComment.epics.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import { batchActions } from 'redux-batched-actions';
+import { ofType } from 'redux-observable';
+import { map } from 'rxjs/operators';
+import uuid from 'd2-utilizr/lib/uuid';
+import { moment } from 'capture-core-utils/moment/momentResolver';
+import { convertValue as convertListValue } from '../../converters/clientToList';
+import { dataElementTypes } from '../../metaData';
+import { actionTypes, batchActionTypes, startAddNoteForEvent } from './WidgetEventComment.actions';
+
+import {
+    addEventNote,
+    removeEventNote,
+} from '../Pages/ViewEvent/ViewEventComponent/editEvent.actions';
+
+import {
+    addNote,
+    removeNote,
+} from '../DataEntry/actions/dataEntry.actions';
+import { getCurrentUser } from '../../d2/d2Instance';
+
+export const addNoteForEventEpic = (action$: InputObservable, store: ReduxStore) =>
```

Comment:
Hey, 
I think we can take advantage of this refactor of `Comment` Widget and get rid of the `epics`, `actions`, and `reducers`. You can use instead the [DHIS2 application runtime hooks](https://runtime.dhis2.nu/#/hooks/). WHDY?
Thanks!

Reply:
hey @simonadomnisoru I don't know that we have plan to get rid of the epics and actions, they are still in used when get from the widget